File: shared/src/commonMain/kotlin/com/walletline/presentation/model/WalletUiItem.kt
package com.walletline.presentation.model

import com.walletline.data.util.randomUUID

data class WalletUiItem(
    val id: String = randomUUID(),
    val title: String,
    val balance: String,
    val description: String?,
    val lines: List<WalletLineUiItem>
)

File: iosApp/iosApp/presentation/model/WalletUiItem.swift
//
//  WalletAccount.swift
//  iosApp
//
//  Created by Mostafa Hosseini on 6/18/23.
//  Copyright © 2023 orgName. All rights reserved.
//

import Foundation

struct WalletUiItem: Hashable, Identifiable {
    let id: String
    let title: String
    let balance: Decimal
    let description: String?
    let lines: [WalletLineUiItem]
    let transactions: [TransactionUiItem]
}
